.clang-tidy

```diff
@@ -44,10 +48,51 @@ Checks: >
 # To turn all warnings into errors, simply write '*'
 # in the first line after WarningsAsErrors
 WarningsAsErrors:  >
-    *
+    bugprone-assert-side-effect,
```

Comment:
Isn't there an easier way to turn all but a single check into an error? Something like `*,-foobar`?

Reply:
That's indeed much better and it works.